### PR TITLE
DCOS 1.13.7 Release Notes

### PR DESCRIPTION
**Pull request by cmastr**
## Jira Ticket
Before creating this pull request, please make sure you have a separate ticket for the docs team to track their work to review and merge, even for minor edits. If you need assistance, ask in the #doc-team channel on slack.

https://jira.mesosphere.com/browse/DCOS-62144

## Description of changes being made

New release notes for 1.13.7 

## Checklist
- [x] Make sure you change all affected versions (e.g. 1.10, 1.11, 1.12, 1.13).
- [x] Test all commands and procedures where applicable.
- [x] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

File: pages/mesosphere/dcos/1.13/release-notes/1.13.7/index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+- [Mesos] Support quoted realms in WWW-Authenticate header. (DCOS-61529)
+- The Service tab in DC/OS now displays services, as expected. (DCOS-61439)
+- DC/OS can now ping between nodes, as expected. (DCOS-60956)
+- The refresh rate for the DC/OS GUI has been decreased. (DCOS-60344)
+- Fixed an issue where (DCOS-60342)
+- Fixed an issue where (DCOS-59959)
+- DC/OS overlay networks should be compared by-value. (DCOS_OSS-5620)
+- Drop labels from Lashup's kv_message_queue_overflows_total metric. (DCOS_OSS-5634)
+- Reserve all agent VTEP IPs upon recovering from replicated log. (DCOS_OSS-5626)
+- Set network interfaces as unmanaged for networkd only on coreos. (DCOS-60956)
```
Comment (cneth) [inline pages/mesosphere/dcos/1.13/release-notes/1.13.7/index.md]: "Set network interfaces as unmanaged for networkd only on CoreOS.  This fixes an issue that could prevent the overlay network from working properly in some situations."

File: pages/mesosphere/dcos/1.13/release-notes/1.13.7/index.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+# Release summary
+DC/OS is a distributed operating system that enables you to manage resources, application deployment, data services, networking, and security in an on-premise, cloud, or hybrid cluster environment.
+
+# Fixed and Improved Issues in DC/OS 1.13.7
+
+<!-- The issues that have been fixed and improved in DC/OS 1.13.7 are grouped by feature, functional area, or component.  -->
+
+- Marathon now bundles multiple concurrent authentication requests for the same identity to a single request, reducing thundering-herd effects on the underlying DC/OS authentication infrastructure. (DCOS-62006)
+- Use Golang 1.10.8 to build CockroachDB. (DCOS-61502)
+- [Mesos] Support quoted realms in WWW-Authenticate header. (DCOS-61529)
```
Comment (cneth) [inline pages/mesosphere/dcos/1.13/release-notes/1.13.7/index.md]: Suggestion: "[Mesos] Support quoted realms in WWW-Authenticate header.   This fixes a problem that prevented the UCR from pulling from Docker registries configured with authentication that used quoted realms."

File: pages/mesosphere/dcos/1.13/release-notes/1.13.7/index.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+- The Service tab in DC/OS now displays services, as expected. (DCOS-61439)
+- DC/OS can now ping between nodes, as expected. (DCOS-60956)
```
Comment (cneth) [inline pages/mesosphere/dcos/1.13/release-notes/1.13.7/index.md]: Please delete this one, this is mentioned below as the last item.

File: pages/mesosphere/dcos/1.13/release-notes/1.13.7/index.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+- The refresh rate for the DC/OS GUI has been decreased. (DCOS-60344)
+- Fixed an issue where (DCOS-60342)
```
Comment (SarahStegall) [inline pages/mesosphere/dcos/1.13/release-notes/1.13.7/index.md]: Suggestion: "Added modal dialogue to select refresh rate to UI configuration screen. (DCOS-60342)"

File: pages/mesosphere/dcos/1.13/release-notes/1.13.7/index.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+- Fixed an issue where (DCOS-59959)
```
Comment (SarahStegall) [inline pages/mesosphere/dcos/1.13/release-notes/1.13.7/index.md]: Suggestion: "Fixed issue where updateAllocation() stalls the allocator where there is a large number of reservations on an agent. (DCOS-59999, MESOS-10015"

Though I'm not sure we always add the Mesos ticket...

File: pages/mesosphere/dcos/1.13/release-notes/1.13.7/index.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+- DC/OS overlay networks should be compared by-value. (DCOS_OSS-5620)
```
Comment (SarahStegall) [inline pages/mesosphere/dcos/1.13/release-notes/1.13.7/index.md]: I believe this should be "by value", without a hyphen.

File: pages/mesosphere/dcos/1.13/release-notes/1.13.7/index.md (hunk 1 above)
Comment (SarahStegall) [inline pages/mesosphere/dcos/1.13/release-notes/1.13.7/index.md]: Change "coreos" to "CoreOS".

File: pages/mesosphere/dcos/1.13/release-notes/1.13.7/index.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (cmastr) [reply pages/mesosphere/dcos/1.13/release-notes/1.13.7/index.md]: I just found out that this was not added during this release, so will take this out.

Comment (cmastr) [reply pages/mesosphere/dcos/1.13/release-notes/1.13.7/index.md]: This was not added during the 1.13.7 release, so removing.

File: pages/mesosphere/dcos/1.13/release-notes/1.13.7/index.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+- Drop labels from Lashup's kv_message_queue_overflows_total metric. (DCOS_OSS-5634)
```
[the remark this answers is not in the document]
Comment (cmastr) [reply pages/mesosphere/dcos/1.13/release-notes/1.13.7/index.md]: didn't want to use "crash" but used some suggestions.

File: pages/mesosphere/dcos/1.13/release-notes/1.13.7/index.md (hunk 5 above)
Comment (vinodkone) [reply pages/mesosphere/dcos/1.13/release-notes/1.13.7/index.md]: @SarahStegall  Sounds good to me. No need for Mesos ticket.

[the remark this answers is not in the document]
Comment (cmastr) [reply pages/mesosphere/dcos/1.13/release-notes/1.13.7/index.md]: removed.

Comment (cmastr) [reply pages/mesosphere/dcos/1.13/release-notes/1.13.7/index.md]: Thanks all! Removing this line.

**Pull request merged**